src/notation/qml/MuseScore/NotationScene/ConcertPitchControl.qml

```diff
@@ -33,10 +33,13 @@ FlatButton {
 
     orientation: Qt.Horizontal
     transparent: true
+    margins: 4
 
     contentItem: Row {
         spacing: 6
 
+        width: implicitWidth + 4 // some extra right padding
```

Comment:
As far as I understand, you can rewrite your code like this:

width: implicitWidth + root.margins

Reply:
Semantically, not really, I think. That extra padding is not necessarily meant to be the same as the normal margin.